.github/workflows/generate-stats-prod.yml

```diff
@@ -0,0 +1,18 @@
+name: Generate Stats for Prod
+
+on:
+  workflow_dispatch:
+
+jobs:
```

Comment:
Just a note, don't think you need to address it. There's a lot of duplication between the test/prod jobs that could be annoying if we need to change things in the future. What do you think of moving the setting of `EARTHDATA_USERNAME`, `EARTHDAT_PASSWORD` ..., `SNS_TOPIC_ARN` into a `with` block in the new action.yml?

Reply:
Fair point. I don't think this will ever change again, since the stats workflow was more useful several months ago when Franz was specifically interested in the output, and this PR is sort of meant to retire the workflow but leave it in place in case we want to trigger it manually. I'm going to go ahead and merge just to get this done, but will keep it in mind for future workflows like this.